docs/content/quickstart/_index.md

```diff
@@ -345,4 +345,6 @@ You now know how to install and configure the Porter Operator. The project is st
 [install-porter]: https://github.com/getporter/porter/releases?q=v1.0.0&expanded=true
 [KinD]: /best-practices/kind/
 [Minikube]: /best-practices/minikube/
+[configure-kind]: https://github.com/getporter/porter/blob/main/docs/content/docs/integrations/kind.md
```

Comment:
You should be able to link to these by pointing them to `/integrations/kind.md` and `integrations/minikube.md` does that work?

Reply:
Those files live in the main porter repo. Will it work?

Reply:
Pretty sure they will, don't best-practices also live in main porter repo?
I think the website pulls the docs from this repo as well and "merges" them together as one

Reply:
You were right, btw - but I ended up grabbing this in my bigger issue. Sorry :(